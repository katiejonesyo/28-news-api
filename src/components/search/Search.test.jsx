import React, { Component } from 'react'

export default class Search.test extends Component {
    render() {
        return (
            <div>
                
            </div>
        )
    }
}
