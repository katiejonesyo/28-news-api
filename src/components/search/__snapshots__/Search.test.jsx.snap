// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search component renders Search 1`] = `
<DocumentFragment>
  <form
    style="text-align: center;"
  >
    <input
      placeholder="Search articles"
    />
  </form>
</DocumentFragment>
`;
